### DISCUSSION TITLE
How to pack more DLLs into tauri bundle?

### DISCUSSION THREAD

**Discussion by datarankio** [tauri-apps/tauri#2662] [Q&A]
Hi Friends, I'm learning to play Tauri on windows and I've been able to generate bundle file, which is able to be installed onto other new machines, but failed to run the app. eventually I figured out there're 3 DLLs missing, msvcp140.dll/vcruntime140.dll/vcruntime140_1.dll. by manually copy/paste these files into the app destination folder, it works. However, I want to know is there any workaround how can I include these a few missing DLLs when I generate the bundle?

Thanks & Regards, 
Martin.

**Comment by datarankio** [comment]
Hi Friends,
I'm able to fix the issue by myself with below workaround:
1, copy the 3 DLLs into src-tauri folder,
2, edit tauri.conf.json and put below lines:
 "resources": [
        "./msvcp140.dll",
        "./vcruntime140.dll",
        "./vcruntime140_1.dll"
      ],

Then the bundle will include these DLLs and the installation will work normally.

Thanks & Regards,
Martin.

**Reply by fy2462** [reply]
I found `files` in `bundle.deb`,  it can copy the files you spacify on Linux.
I don't find the filed in `bundle.windows`, but I think you can copy them in `wxs` file, 
Example:
https://github.com/kimlimjustin/xplorer/blob/master/src-tauri/installer.wxs

**Comment by elanticrypt0** [comment]
Thank you
